docs/api/protocol.md

```diff
@@ -87,7 +87,7 @@ module gets emitted.
     * `method` String
     * `uploadData` [UploadData[]](structures/upload-data.md)
   * `callback` Function
-    * `filePath` String (optional)
+    * `filePath` ( String | {path:string} | number | {error:any} ) (optional)
```

Comment:
Typescript style definitions don't go here as they can't be interpreted by the docs linter.  As much as it seems weird we must make two new "structure" docs here.  `PathObject` and `ErrorObject`.  Each of which formally documenting the structure outlined here

Reply:
@3n-mb wanna take a shot at this using the name `ErrorObject`?

Reply:
@zeke anything here is good, as it is an improvement over an original, that had only `string` as a type.

Reply:
@3n-mb can you create the two new docs/api/structures/ files that @MarshallOfSound alluded to above?

Reply:
@zeke you better shoot me. I am totally of context, especially not being familiar with this docs linter.